src/shady/api/generator_grammar.c

```diff
@@ -160,13 +160,11 @@ void generate_node_ctor(Growy* g, json_object* src, json_object* nodes) {
         shd_growy_append_formatted(g, " {\n");
         shd_growy_append_formatted(g, "\tNode node;\n");
         shd_growy_append_formatted(g, "\tmemset((void*) &node, 0, sizeof(Node));\n");
-        shd_growy_append_formatted(g, "\tnode = (Node) {\n");
-        shd_growy_append_formatted(g, "\t\t.arena = arena,\n");
-        shd_growy_append_formatted(g, "\t\t.tag = %s_TAG,\n", name);
+        shd_growy_append_formatted(g, "\tnode.arena = arena;\n");
```

Comment:
What's the purpose of this change ?

Reply:
The project I'm using it in is compiling with C++20 which throws an error if not all fields are included in such an initializer list statement. It seems to only trigger in the generated code and I figured it was easier to simply use a syntax that causes less friction between different compilers.

Reply:
I see. I've been doing a lot of deep refactoring work, and I've not looked at windows builds in a hot second. The C++ header inclusion issue is something I'll try to keep in mind!